### PR TITLE
Update Triage documentation to reflect 2019-07-03 meeting

### PR DESCRIPTION
**Pull request by chocol4te**
On 2019-07-03 there was a Discord meeting on `#triage-wg` that discussed some changes to the triage process. This PR updates the Forge documentation to reflect said changes.

### CODE REVIEW

**Comment by chocol4te** [comment]
I'm also noticing some styling issues regarding line width. Files seem to be at 70 characters, others 100, others have no limit, and some have a mixture of the above. I'm happy to go through and fix this if a agreement over styling can be made (either no wrapping, or hard wrapping with a consistent length).

**Comment by Mark-Simulacrum** [comment]
A separate PR hard-wrapping at width 80 would be great.

**Comment by chocol4te** [comment]
On it now :)

**Comment by chocol4te** [comment]
I need to rebase and then this PR will be ready for review.

**Comment by chocol4te** [comment]
A reviewer, a reviewer, my kingdom for a reviewer!

File: triage-procedure.md [hunk 1]
```diff
@@ -75,70 +78,59 @@ At this point, all PRs must have a tag applied.
...
-  subteam on GitHub and ask for a new reviewer.
-- If the PR is more than 9 days old, in addition to the above steps, change the
-  tag to S-waiting-on-team
+If the review is complete the label should be changed from `S-waiting-on-review`
+to `S-waiting-on-author`.
 
-If the **review is incomplete**:
+Otherwise, the reviewer should be pinged. It is a good idea to start the message
+with "Ping from Triage..." so that the concerned parties know it is coming from
+the triage team, and the message should be asing the reviewer to either review
```
Comment (jonas-schievink) [inline triage-procedure.md]:
```suggestion
the triage team, and the message should be asking the reviewer to either review
```

File: triage-procedure.md [hunk 2]
```diff
@@ -75,70 +78,59 @@ At this point, all PRs must have a tag applied.
...
 
-- If more than 3 days since the last reviewer comment, ping the reviewer on IRC
-  and on GitHub.
-- If more than 6 days since the last reviewer comment, also ping the subteam on
-  GitHub.
-- If more than 9 days since the last reviewer comment, also change the tag to
-  S-waiting-on-team
+The `r?` command is needed to override a reviewer, however not all triagers will
+have sufficient permissions. In this case sending a message to the `#triage-wg`
+Discord or pinging @dpc will be necessary.
```
Comment (jonas-schievink) [inline triage-procedure.md]:
```suggestion
Discord or pinging @Dylan-DPC will be necessary.
```

File: triage-procedure.md [hunk 3]
```diff
@@ -27,18 +25,20 @@ title: Triage Procedure
...
+[s-waiting-on-team]:https://github.com/rust-lang/rust/pulls?q=is%3Aopen+is%3Apr+label%3AS-waiting-on-team+sort%3Aupdated-desc
+[s-waiting-on-bors]:https://github.com/rust-lang/rust/pulls?q=is%3Aopen+is%3Apr+label%3AS-waiting-on-bors+sort%3Aupdated-asc
+[s-waiting-on-crater]:https://github.com/rust-lang/rust/pulls?q=is%3Aopen+is%3Apr+label%3AS-waiting-on-crater+sort%3Aupdated-asc
+[s-waiting-on-bikeshed]:https://github.com/rust-lang/rust/pulls?q=is%3Aopen+is%3Apr+label%3AS-waiting-on-bikeshed+sort%3Aupdated-asc
+[s-waiting-on-perf]:https://github.com/rust-lang/rust/pulls?q=is%3Aopen+is%3Apr+label%3AS-waiting-on-perf+sort%3Aupdated-asc
+[s-blocked]:https://github.com/rust-lang/rust/pulls?q=is%3Aopen+is%3Apr+label%3AS-blocked+sort%3Aupdated-asc
+[s-blocked-closed]:https://github.com/rust-lang/rust/pulls?q=is%3Apr+label%3AS-blocked-closed+sort%3Aupdated-asc
+[s-inactive-closed]:https://github.com/rust-lang/rust/pulls?q=is%3Apr+label%3AS-inactive-closed+sort%3Aupdated-asc
+[crater]:https://github.com/rust-lang-nursery/crater
+[bors]:https://github.com/graydon/bors
```
Comment (jonas-schievink) [inline triage-procedure.md]: This isn't what we're using anymore

```suggestion
[bors]:https://github.com/rust-lang/homu
```

**Comment by chocol4te** [comment]
Thanks @jonas-schievink! :D

**Comment by jonas-schievink** [comment]
Okay, looks good to me now. Maybe @Dylan-DPC can confirm that this is correct?

**Pull request merged**